tensorflow/core/kernels/data/dataset_test_base.h

```diff
@@ -95,6 +108,17 @@ class DatasetOpsTestBase : public ::testing::Test {
     return Status::OK();
   }
 
+  // Creates a new TensorSliceDataset op kernel.
+  Status CreateTensorSliceDatasetKernel(
+      const StringPiece& node_name, const DataTypeVector& dtypes,
+      const std::vector<PartialTensorShape>& shapes,
+      std::unique_ptr<OpKernel>* tensor_dataset_kernel);
+
+  // Creates a new TensorSliceDataset.
+  Status CreateTensorSliceDataset(const StringPiece& node_name,
+                                  std::vector<Tensor>& components,
```

Comment:
Pass a pointer instead of a non-const reference. https://google.github.io/styleguide/cppguide.html#Reference_Arguments